test/stream/api/source/version-detail-404/003-patch-settings/request.json

```diff
@@ -0,0 +1,21 @@
+{
+  "path": "/v1/branches/default/sources/my-source/settings",
```

Comment:
`version-detail-404` can be simplified. You don't need to patch settings, you aren't using the new version in the test anyway.

Reply:
[0377fac](https://github.com/keboola/keboola-as-code/pull/2087/commits/0377facfe1520062da1e095c3d68e991ce59c69e)